docs/source/features/dataset_management.rst

```diff
@@ -0,0 +1,151 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+####################
+ Dataset Management
+####################
+
+:class:`~graviti.manager.dataset.Dataset` is the most basic concept in Graviti SDK. Almost all
+operations require a dataset first. Of course, it is necessary to initialize a
+:class:`~graviti.workspace.Workspace` before managing the dataset:
+
+.. code:: python
+
+   from graviti import Workspace
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+
+******************
+ Create a Dataset
+******************
+
+SDK provides method :func:`~graviti.manager.dataset.DatasetManager.create` to support creating
```

Comment:
what is the difference between sphinx `:func:` and `:meth:`

Reply:
no difference here
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-roles
https://stackoverflow.com/questions/34848253/whats-the-difference-in-behaviour-between-func-and-meth-roles-in-python-sph

Reply:
will it be better if we use `:meth:` here?

Reply:
OK